runtime/near-vm-runner/src/tests.rs

```diff
@@ -70,6 +72,7 @@ fn prepaid_loading_gas(bytes: usize) -> u64 {
 /// memory must be configured for indices `0..WASM_PAGE_SIZE` to be valid.
 ///
 /// Panics if any of the tests fails.
+#[cfg(all(feature = "wasmer-runtime", target_arch = "x86_64"))]
```

Comment:
This test function is intended to be applicable for all runtimes we implement. If the `PAGE` constant in there is causing problems, I think it'd make sense to hardcode its value (`64 * 1024` IIRC)

Reply:
It is the PAGE constant, I set it to the value from: 
https://docs.rs/wasmer/latest/wasmer/constant.WASM_PAGE_SIZE.html .
In cargo build this code generates unused code warnings so I added allow(dead_code). It solves the problem but I feel like if we could find the right feature/ target_arch that would be a cleaner.

Reply:
The webassembly page size is specified by the webassembly specification, so there is really no strong reason to depend on any particular runtime to provide this value over specifying one in-line.